### ISSUE TITLE
Wording of consensus based argument against mandatory incubation

### ISSUE THREAD

**Issue by wcarr** [w3c/standards-track#7]
In list after "Arguments against mandating using W3C Community Groups to incubate proposals until they met empirical criteria included"

> If the CG did not work by consensus, the de facto standard might be imposed by a single strong editor or cabal of the top browsers

I think that wording minimizes the problem.  I suggest rewording it as:

> CG rules allow them to be controlled by a single person, while W3C WGs guarantee fairness.  Requiring external incubation (somewhere like a CG) could lead to most key technical decisions being made in forums dominated by a few, reducing the level playing field provided by W3C to when implementation makes it too late.

**Comment by michaelchampion** [comment]
I've updated the "arguments against" with something close to wcarr's suggested wording: - Requiring external incubation in a venue that doesn't require broad consensus
 could lead to key technical decisions being made in forums dominated by a few, 
 with "level playing field" discussion beginning only when implementation momentum makes change difficult.
